Automattic/vip-go-mu-plugins
# Update ElasticPress submodule to latest

## Description

Brings in a fix for trashed posts still appearing in the index

## Checklist

Please make sure the items below have been covered before requesting a review:

- n/a This change has relevant documentation additions / updates (if applicable).

## Steps to Test

Has been tested separately, but good to just test general search functionality. Particularly, trashing a post (on sites that don't have protected content indexed) should no longer leave that post in the index (can verify with `wp vip-search health validate-contents`.
